libraries/lib-wave-track/WaveTrack.cpp

```diff
@@ -1421,7 +1421,10 @@ void WaveTrack::BlindlyCopyTrackContents(double t0, const WaveTrack& other)
 void WaveTrack::BlindlyMergeClipFromOtherTrackInto(
    double t0, WaveClip& myClip, const WaveClip& otherTrackClip)
 {
-   myClip.Paste(t0 - myClip.GetPlayStartTime(), otherTrackClip);
+   // No need for a progress bar, since clips must have equal stretch ratios:
+   assert(
+      fabs(myClip.GetStretchRatio() - otherTrackClip.GetStretchRatio()) < 1e-6);
```

Comment:
Another tolerance

Reply:
Removed.

Reply:
> Removed.

NO!

Reply:
Use a tolerance!  But factor some common function or constant to decide what the tolerance is.